lib_nbgl/include/nbgl_use_case.h

```diff
@@ -40,7 +40,11 @@ extern "C" {
 /**
  *  @brief maximum number of lines for value field in details pages
  */
+#ifdef TARGET_STAX
 #define NB_MAX_LINES_IN_DETAILS 12
+#else  // TARGET_STAX
+#define NB_MAX_LINES_IN_DETAILS 11
```

Comment:
I guess the below defines could be changed too? (`NB_MAX_LINES_IN_REVIEW` / `NB_MAX_DISPLAYED_PAIRS_IN_REVIEW`)

Reply:
To be tuned later when the copy will be dry